src/test/golang/pgadapter_pgx_tests/pgx.go

```diff
@@ -334,6 +335,112 @@ func TestPrepareSelectStatement(connString string) *C.char {
 	return nil
 }
 
+//export TestInsertBatch
+func TestInsertBatch(connString string) *C.char {
```

Comment:
This now actually uses an `ExecuteBatchDml` request instead of a collection of separate `ExecuteSql` requests.